pandas/tests/io/excel/test_readers.py

```diff
@@ -988,3 +988,14 @@ def test_conflicting_excel_engines(self, read_ext):
         with pd.ExcelFile("test1" + read_ext) as xl:
             with pytest.raises(ValueError, match=msg):
                 pd.read_excel(xl, engine="foo")
+
+    def test_excel_read_binary(self, engine, read_ext):
+        # GH 15914
+        expected = pd.read_excel("test1" + read_ext, engine=engine)
+
+        with open("test1" + read_ext, "rb") as f:
+            data = f.read()
+            f.close()
```

Comment:
why is this necessary?

Reply:
it's not, thank you as I didn't remember this